libevmasm/Assembly.h

```diff
@@ -78,9 +78,9 @@ class Assembly
 	/// Adds a subroutine to the code (in the data section) and pushes its size (via a tag)
 	/// on the stack. @returns the pushsub assembly item.
 	AssemblyItem appendSubroutine(AssemblyPointer const& _assembly) { auto sub = newSub(_assembly); append(newPushSubSize(size_t(sub.data()))); return sub; }
-	void pushSubroutineSize(size_t _subRoutine) { append(newPushSubSize(_subRoutine)); }
+	void pushSubroutineSize(u256 _subRoutine) { append(newPushSubSize(_subRoutine)); }
```

Comment:
Why were all of these changed to `u256`?

Reply:
I changed to u256 so it can accept multiple 8 bit ids. Leftmost byte is how many ids we have starting from rightmost bit. That way we can go into sub objects in multiple levels.